reference/v8.json

```diff
@@ -646,7 +646,7 @@
         "height"
       ],
       "default": "none",
-      "doc": "Position and scale an icon by the its corresponding text.",
+      "doc": "Position and scale of an icon relative to the dimensions of text in `text-field`.",
```

Comment:
I find that enumerated properties are best documented by providing one sentence about what the property does in general, and then descriptions of the effects of each value. For example:

> Permits scaling the icon such that it fits the associated text. The value `none` performs no scaling. The values `width` and `height` produce an icon scaled in the x- or y-dimension, respectively, such that the text fits inside the icon. The value `both` produces scaling in both dimensions.